src/chaos.js

```diff
@@ -52,4 +91,64 @@ export class KubernetesClient {
         this.kubernetes.namespaces.delete(randomNamespace.name)
         return randomNamespace.name
     }
-}
\ No newline at end of file
+}
+
+export class StressNodesAttack {
+    constructor(client) {
+        this.kubernetes = client
+        this.name = 'stress-nodes'
+        this.namespace = 'default'
+        this.cpu = '0'
+        this.io = '0'
+        this.vm = '0'
+        this.vmBytes = '256MB'
+        this.timeout = '10'
+    }
+
+    inNamespace(namespace) {
+        this.namespace = namespace
+        return this
+    }
+    inNodes(nodes) {
+        this.nodes = nodes
+        return this
+    }
+    withName(name) {
+        this.name = name
+        return this
+    }
+    withCPU(cpu) {
+        this.cpu = cpu
+        return this
+    }
+    withIO(io) {
+        this.io = io
+        return this
+    }
+    withVM(vm) {
+        this.vm = vm
+        return this
+    }
+    withVMBytes(vmBytes) {
+        this.vmBytes = vmBytes
+        return this
+    }
+    withTimeout(timeout) {
+        this.timeout = timeout
+        return this
+    }
+    start() {
+        this.kubernetes.runJobOnNodes(
+            this.namespace,
+            this.name,
+            this.nodes,
+            'progrium/stress',
+            ['stress', '--cpu', this.cpu, '--io', this.io, '--vm', this.vm, '--vm-bytes', this.vmBytes, '--timeout', this.timeout]
+        )
+        return this.name
+    }
+    clean() {
+        this.kubernetes.cleanJobs(this.namespace, this.name)
+        return this.name
```

Comment:
Not sure if I understand the semantic (and more importantly, the usage) of the `clean` function. If it only clean jobs, it could return the instance and allow another `start`. If this is not the intention, then maybe returning a new instance with default values.